packages/build/config/mocha.opts

```diff
@@ -1,4 +0,0 @@
---require source-map-support/register
---recursive
---exit
---reporter dot
```

Comment:
I believe we have two Mocha configurations:
1. The config in `packages/build` that's primarily used inside loopback-next monorepo. It's important to include `--exit` and `--reporter dot` options.
2. The template in `packages/cli/generators/project/templates/test/mocha.opts` that's used when scaffolding new LB4 project.

IIUC, your pull request is correctly reworking the template from `mocha.opts` to `.mocharc.json`, but completely removing the config used by loopback-next. Please preserve both sets of configuration. Maybe you just forgot to commit the file `packages/build/config/.mocharc.json`? I see you are referring to it in multiple places, but don't see that file in the pull request.

While we are discussing this part, can we remove the dot prefix from the filename please, to make the file easier to find? Dot files are considered as "hidden" on Unix-like systems. It makes a lot of sense to use a dot file in project root (`.mocharc.json`), but I find it counter-productive to use a dot file elsewhere in project directory tree.

I am proposing the following filename:

```
packages/build/config/mocharc.json
```

Reply:
@raymondfeng 

> Is .mocharc.json required now? IMO, it should be optional.

It's not required *unless* the path is explicitly given via `--config <path>`.

Reply:
I named it `.mocharc.json` because the other dotfiles in that directory begin with a dot (e.g., `.npmrc`, `.prettierrc`).

Reply:
I think I'd prefer to keep the name as it is for consistency and to avoid confusion between `.mocharc.json` and `mocharc.json`?

Reply:
> I named it `.mocharc.json` because the other dotfiles in that directory begin with a dot (e.g., `.nycrc`, `.prettierrc`).  
> I think I'd prefer to keep the name as it is for consistency and to avoid confusion between `.mocharc.json` and `mocharc.json`?

Fair enough 👍

Reply:
I believe this part was not addressed yet:

IIUC, your pull request is correctly reworking the template from `mocha.opts` to `.mocharc.json`, but completely removing the config used by loopback-next. Please preserve both sets of configuration. Maybe you just forgot to commit the file `packages/build/config/.mocharc.json`? I see you are referring to it in multiple places, but don't see that file in the pull request.